run_build_impl.sh

```diff
@@ -48,19 +48,10 @@ echo "-----------------------------"
 REP=$(find . -maxdepth 3 -type d -name .git -a \( -path "./$DEPS/*" -prune -o -print -quit \) )
 if [ -n "${REP}" ]; then
   REP=$(dirname "${REP}")
-  cd "${REP}" && repo_url_self=$(git config --get remote.origin.url)
-else
-  echo "ERROR: Could not find repository to run Jenkins independent refetch."
-  exit 1
+  repo_url_self=$(cd "${REP}" && git config --get remote.origin.url)
```

Comment:
Hm, I'm not sure what you mean. Is there a typo? Did you mean "can't be removed"? 
In that case : yes, yes.